### PR TITLE
doc: add note about endpoint.name==service.name

### PR DESCRIPTION
**Pull request by Shabirmean**
#### Description

- For Services without selectors, there is no mention of the requirement that the name of the associated Endpoint object must be the same as the Service name. 
- Thus, I have added that note.

### CODE REVIEW

File: content/en/docs/concepts/services-networking/service.md
Comment (sftim) [inline content/en/docs/concepts/services-networking/service.md:206]: - Kubernetes doesn't have an API type “Endpoint”; did you mean “[Endpoints](https://kubernetes.io/docs/reference/kubernetes-api/service-resources/endpoints-v1/)”
- how does this requirement apply to the [EndpointSlice](https://kubernetes.io/docs/reference/kubernetes-api/service-resources/endpoint-slice-v1/) API?


How about:
```suggestion
When you create an [Endpoints](docs/reference/kubernetes-api/service-resources/endpoints-v1/)
object for a Service, you set the name of the new Endpoints to be the same as the
name of the Service.
```

Comment (sftim) [reply content/en/docs/concepts/services-networking/service.md:206]: You could also put a comment into the example Endpoints manifest @Shabirmean

Comment (Shabirmean) [reply content/en/docs/concepts/services-networking/service.md]: Thank you for the review @sftim. I've adopted the changes. Also I think EndpointSlices don't have this requirement. The association is done via adding an annotation.

File: content/en/docs/concepts/services-networking/service.md
Comment (shannonxtreme) [inline content/en/docs/concepts/services-networking/service.md:208]: I would say "Endpoints object" in every mention of the object. Otherwise it reads a bit strangely. Nit, really.

Comment (Shabirmean) [reply content/en/docs/concepts/services-networking/service.md]: Done in https://github.com/kubernetes/website/pull/33197/commits/65dbc312c73cbe7873dd63aac60d9c2d0198052c

File: content/en/docs/concepts/services-networking/service.md
Comment (divya-mohan0209) [inline content/en/docs/concepts/services-networking/service.md:207]:
```suggestion
When you create an [Endpoint](docs/reference/kubernetes-api/service-resources/endpoints-v1/)
```

File: content/en/docs/concepts/services-networking/service.md
Comment (divya-mohan0209) [inline content/en/docs/concepts/services-networking/service.md:208]:
```suggestion
object for a Service, you set the name of the new object to be the same as that
```

File: content/en/docs/concepts/services-networking/service.md
Comment (divya-mohan0209) [inline content/en/docs/concepts/services-networking/service.md:209]:
```suggestion
of the Service.
```

**Review by divya-mohan0209 — CHANGES_REQUESTED** [review]

**Comment by divya-mohan0209** [comment]
While I understand the object name is "Endpoints", elsewhere in the same doc it has been amended to be singular for better readability. This is not a blocking nit but will help maintain consistency across the doc.

**Comment by sftim** [comment]
We should **always** use “Endpoints” when talking about the Endpoints API. When we're talking about endpoints as a concept, it's OK to write “endpoint” or “endpoints”.

PR https://github.com/kubernetes/website/pull/30817 also does some tidying around this BTW.

File: content/en/docs/concepts/services-networking/service.md
Comment (sftim) [reply content/en/docs/concepts/services-networking/service.md:209]: “the same as the of the Service” doesn't look idiomatic to me @divya-mohan0209 - is that definitely the change you wanted to suggest?

Comment (divya-mohan0209) [reply content/en/docs/concepts/services-networking/service.md:209]: I meant to suggest "the same as that of the Service".

File: content/en/docs/concepts/services-networking/service.md
Comment (Shabirmean) [reply content/en/docs/concepts/services-networking/service.md:207]: Avoiding suggestion as per comment https://github.com/kubernetes/website/pull/33197#issuecomment-1113104092

**Comment by Shabirmean** [comment]
Hello team,

Whats the process to get this merged? Is there something else on my part to be done before we can merge it?

**Comment by sftim** [comment]
> Whats the process to get this merged?

The process is a [pull request review](https://kubernetes.io/docs/contribute/review/).

> Is there something else on my part to be done before we can merge it?

At the moment, you're waiting for further feedback from a reviewer.

--- 

https://github.com/kubernetes/website/pull/33197#pullrequestreview-957842016 marked this as _informally_ “LGTM” but nobody has provided a signoff /lgtm.

@divya-mohan0209 what further review were you looking for here? A tech review perhaps, from the relevant SIG? Any concerns about other aspects of the PR?

**Pull request merged**
